File: app/src/androidTest/java/co/ericp/freewayforecast/HomeScreenUITest.java
package co.ericp.freewayforecast;

import android.support.test.filters.LargeTest;
import android.support.test.rule.ActivityTestRule;
import android.support.test.runner.AndroidJUnit4;

import org.junit.Rule;
import org.junit.Test;
import org.junit.runner.RunWith;

import static android.support.test.espresso.Espresso.*;
import static android.support.test.espresso.action.ViewActions.*;
import static android.support.test.espresso.assertion.ViewAssertions.*;
import static android.support.test.espresso.matcher.ViewMatchers.*;

@RunWith(AndroidJUnit4.class)
@LargeTest
public class HomeScreenUITest {

    @Rule
    public ActivityTestRule<MainActivity> mActivityRule = new ActivityTestRule<>(
            MainActivity.class);

    @Test
    public void displaysInputAtFirst() {
        onView(withId(R.id.input_origin)).check(matches(isDisplayed()));
    }
}

File: app/src/androidTest/java/co/ericp/freewayforecast/HomeScreenUITest.kt
package co.ericp.freewayforecast

import android.support.test.filters.LargeTest
import android.support.test.rule.ActivityTestRule
import android.support.test.runner.AndroidJUnit4

import org.junit.Rule
import org.junit.Test
import org.junit.runner.RunWith

import android.support.test.espresso.Espresso.*
import android.support.test.espresso.action.ViewActions.*
import android.support.test.espresso.assertion.ViewAssertions.*
import android.support.test.espresso.matcher.ViewMatchers.*

@RunWith(AndroidJUnit4::class)
@LargeTest
class HomeScreenUITest {

    @Rule
    var mActivityRule = ActivityTestRule(MainActivity::class.java)

    @Test
    fun displaysInputAtFirst() {
        onView(withId(R.id.input_origin)).check(matches(isDisplayed()))
    }
}
